.github/workflows/check-size.yml

```diff
@@ -0,0 +1,22 @@
+name: Check binary size
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, labeled, unlabeled]
```

Comment:
Should we also add `ready_for_review`?

Reply:
I don't think that's ideal, because we would want this to run on commits on draft PRs too. It would help identify bloating early on and change the direction of development accordingly. As far as I understand, `ready_for_review` is triggered when a draft PR is converted into a regular one. By then, it might be too late to notify this to the author.

Reply:
`ready_for_review` should not prevent this workflow from running on draft PRs.

Reply:
Ah understood. But that wouldn't change the contents of the PR itself, so wouldn't it be a redundant run? If the workflow has already run on the latest commit before converting from a draft PR to a regular one, then we'll be doing a duplicate run.

Reply:
Running the workflow when converting from a draft PR might be helpful because it would pick up any changes to the main branch since the last run.